### ISSUE TITLE
Mopidy-File doesn't provide images

### ISSUE THREAD

**Issue by kingosticks** [mopidy/mopidy#1895]
Mopidy-File does not yet implement `backend.LibraryProvider.get_images`. [This isn't new](https://github.com/mopidy/mopidy/blob/develop/mopidy/file/library.py#L14] but it's worth tracking and highlight it as a good first issue. Initially it would be sufficient just to get the images from file metadata like the Local backend. 

I added a note in the docs highlighting this limitation.

**Comment by parth-verma** [comment]
Hi, I'd like to work on this.

**Comment by kingosticks** [comment]
Please do. There's info in our documentation about [how to setup a development environment](https://docs.mopidy.com/en/latest/devenv/) and making [pull requests](https://docs.mopidy.com/en/latest/contributing/#pull-request-guidelines).

**Comment by parth-verma** [comment]
So I have submitted a PR, but I want to discuss where do we store the extracted artwork or is there any need to store it at all?

**Comment by kingosticks** [comment]
You have, thanks! As it happens I was reviewing it yesterday but have not quite finished yet. I think the method you've gone for in your PR (storing it) is a good approach.

**Comment by parth-verma** [comment]
Hey, i was trying it out in different frontends and it causes error in browsers in that they cannot display `file://` uris. So instead I changed it so that it now send artwork in base64 encoded format.

**Comment by kingosticks** [comment]
Yes, that is a good point. Clients running on a different machine certainly cannot access local files. The Local backend [serves the artwork it finds](https://github.com/mopidy/mopidy-local/blob/master/mopidy_local/__init__.py#L49-L56).

**Comment by parth-verma** [comment]
I do not have a raspberry pi (zero) so cannot test its performance.

**Comment by kingosticks** [comment]
Sorry if I was unclear, I meant to say that I will do that.

**Comment by parth-verma** [comment]
Would it be a good idea to to return the folder image if the uri is of a directory?

**Comment by kingosticks** [comment]
I guess you could but you may find no clients will ever request that image.
